app/src/main/res/values-ban/strings.xml

```diff
@@ -288,7 +288,7 @@
   <string name="resume">lanjut</string>
   <string name="paused">Karérénang</string>
   <string name="more">Malih</string>
-  <string name="favorites">Senengan</string>
+  <string name="bookmarks">Senengan</string>
```

Comment:
We don't edit string files for all languages manually. Translate updates will do that after your PR is merged to master. So please only edit default strings.xml file.